modules/transactions/src/test/java/org/apache/ignite/internal/tx/storage/state/rocksdb/RocksDbTxStateStorageTest.java

```diff
@@ -43,18 +48,28 @@ public class RocksDbTxStateStorageTest extends AbstractTxStateStorageTest {
     @WorkDirectory
     private Path workDir;
 
+    private final ScheduledExecutorService scheduledExecutor = new ScheduledThreadPoolExecutor(1);
+
+    private final ExecutorService executor = Executors.newFixedThreadPool(1);
```

Comment:
Same here, we can change it to `Executors.newSingleThreadExecutor` for consistency sake

Reply:
fix it